tests/tck/features/fetch/FetchEdges.intVid.feature

```diff
@@ -176,45 +176,45 @@ Feature: Fetch Int Vid Edges
       | serve._src | serve._dst | serve._rank | serve.start_year |
     When executing query:
       """
-      GO FROM hash("NON EXIST VERTEX ID") OVER serve YIELD serve._src AS src, serve._dst AS dst
-      | FETCH PROP ON serve $-.src->$-.dst YIELD serve.start_year, serve.end_year
+      GO FROM hash("NON EXIST VERTEX ID") OVER serve YIELD serve._src AS src, serve._dst AS dst |
+      FETCH PROP ON serve $-.src->$-.dst YIELD serve.start_year, serve.end_year
       """
     Then the result should be, in any order:
       | serve._src | serve._dst | serve._rank | serve.start_year | serve.end_year |
     When executing query:
       """
-      GO FROM hash("NON EXIST VERTEX ID") OVER serve YIELD serve._src AS src, serve._dst AS dst, serve.start_year as start
-      | YIELD $-.src as src, $-.dst as dst WHERE $-.start > 20000
-      | FETCH PROP ON serve $-.src->$-.dst YIELD serve.start_year, serve.end_year
+      GO FROM hash("NON EXIST VERTEX ID") OVER serve YIELD serve._src AS src, serve._dst AS dst, serve.start_year as start |
+      YIELD $-.src as src, $-.dst as dst WHERE $-.start > 20000 |
+      FETCH PROP ON serve $-.src->$-.dst YIELD serve.start_year, serve.end_year
       """
     Then the result should be, in any order:
       | serve._src | serve._dst | serve._rank | serve.start_year | serve.end_year |
     When executing query:
       """
-      GO FROM hash("Marco Belinelli") OVER serve YIELD serve._src AS src, serve._dst AS dst, serve.start_year as start
-                   | YIELD $-.src as src, $-.dst as dst WHERE $-.start > 20000
-                   | FETCH PROP ON serve $-.src->$-.dst YIELD serve.start_year, serve.end_year
+      GO FROM hash("Marco Belinelli") OVER serve YIELD serve._src AS src, serve._dst AS dst, serve.start_year as start |
+      YIELD $-.src as src, $-.dst as dst WHERE $-.start > 20000 |
+      FETCH PROP ON serve $-.src->$-.dst YIELD serve.start_year, serve.end_year
       """
     Then the result should be, in any order:
       | serve._src | serve._dst | serve._rank | serve.start_year | serve.end_year |
 
-  Scenario: Fetch prop Semantic Error
+  Scenario: Fetch prop Error
     When executing query:
       """
       FETCH PROP ON serve hash("Boris Diaw")->hash("Spurs") YIELD $^.serve.start_year
       """
-    Then a SemanticError should be raised at runtime:
+    Then a ExecutionError should be raised at runtime:
```

Comment:
Why change this?

Reply:
validate $^.serve.start_year produce. ExecutorError

Reply:
It's semantic error.

Reply:
use duduceType produce ExecutorError